### DISCUSSION TITLE
Ctrl+c ⌘+c unity

### DISCUSSION THREAD

**Discussion by kaihendry** [debauchee/barrier#1429] [Q&A]
Since I switch between my Archlinux host and Mac machines all day, I'm constantly muddled about copy and pasting between the two operating systems.

Is there some strategy to unify the copy/paste key binds?

**Comment by PhilCS** [comment]
I had the same problem between my Windows host and Mac client, I ended up binding Ctrl to Cmd and the Windows key to Ctrl:

1. Press "Configure Server" on the host
2. Draw the screen icon onto the grid
3. Double-click the new screen and name it with your Mac's hostname, I named mine `Phils-MacBook-Pro.local`
4. Set "Ctrl: Super" and "Super: Ctrl"

Now, the only problem I have is seamlessly mapping Ctrl+Y to Cmd+Shift+Z, which doesn't seem possible... Cmd+Y does all kinds of weird crap on Mac, like deleting text, which is super annoying.

**Reply by kaihendry** [reply]
Didn't quite understand which OS was your host / keyboard.

I'm thinking of trying to rebind my copy/paste keys on Arch atm https://github.com/jordansissel/xdotool/issues/369

**Reply by PhilCS** [reply]
Windows host with Windows USB keyboard

**Reply by konfilios** [reply]
I just successfully applied your suggestion on a linux server (ubuntu 22.04) with macbook client (macOS 13.2) using barrier 2.4.0 and my sanity was restored.

![Screenshot_20230206_150322](https://user-images.githubusercontent.com/1292243/216978409-bab2f82c-b6e4-4b4b-91c1-1ea533e57118.png)

Thank you very much

**Reply by FikemiGitHub** [reply]
Thank you soo much!!!! My sanity is restored🤣🤣🤣

**Comment by feiyuyitiao** [comment]
in barrier client, It wiil be better to add more key mapping settings self defined just like windows PowerToys.
